### PR TITLE
Added Lenovo plugin deployment instructions

### PR DESCRIPTION
**Pull request by sunithasomanna**
Added Lenovo plugin deployment instructions to Getting Started readme

### CODE REVIEW

**Review by shivacharanms — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1458,6 +1459,139 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
+   2. Run the following command to create `lenovoplugin` Helm package at `~/plugins/lenovoplugin`:
+
+          helm package lenovoplugin -d ~/plugins/lenovoplugin
+
+      The Helm package for the Lenovo plugin is created in the tgz format.
+
+7. Save the Lenovo plugin Docker image on the deployment node at `~/plugins/lenovoplugin`.
+
+       docker save lenovoplugin:1.0 -o ~/plugins/lenovoplugin/lenovoplugin.tar
+
```
Comment (sukhwinder20) [inline README.md:1533]: add cd ODIM

Comment (sunithasomanna) [reply README.md:1533]: done

File: README.md [hunk 2]
```diff
@@ -1458,6 +1459,145 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
+         $ kubectl get pods -n odim
+
+      Example output of the Lenovo plugin pod details:
+
+    | NAME                         | READY | STATUS  | RESTARTS | AGE   |
+    | ---------------------------- | ----- | ------- | -------- | ----- |
+    | lenovoplugin-5fc4b6788-2xx97 | 1/1   | Running | 0        | 4d22h |
+
+18. [Add the Lenovo plugin into the Resource Aggregator for ODIM framework](#adding-a-plugin-into-the-resource-aggregator-for-odim-framework).
```
Comment (sukhwinder20) [inline README.md:1597]: Add the example in section Add the Lenovo plugin into the Resource Aggregator for ODIM framework

**Review by sukhwinder20 — CHANGES_REQUESTED** [review]
Lenovoplugin ports information is required to be added

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md:1597]: Done

**Comment by sunithasomanna** [comment]
> Lenovoplugin ports information is required to be added

Done

**Review by sukhwinder20 — APPROVED** [review]

**Pull request merged**